scripts/ci/update-open-pr

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Setup Git
+git config --global user.email "$GIT_USER@users.noreply.github.com"
+git config --global user.name "MesosphereCI Robot"
+git clone --depth 1 -b dcos-ui/latest https://$GIT_USER:$GIT_PASSWORD@github.com/mesosphere/dcos dcos-latest
+ 
+ls -l
+cp -fr buildinfo.json dcos-latest/packages/dcos-ui/buildinfo.json
+cd dcos-latest
```

Comment:
Shellcheck [SC2164](https://github.com/koalaman/shellcheck/wiki/SC2164) tells me that we should do something like 

```sh
cd dcos-latest || exit 1
```

to have the script fail if we can't cd into the directory